samples/stock/demo/compare-etf/demo.js

```diff
@@ -0,0 +1,100 @@
+(async () => {
+
+    const PERIOD_CAP = 31.97,
```

Comment:
Let's set the cap to ~22.1%, so buffered ETF is always below, but the tracked index can go higher